### ISSUE TITLE
Provide support for i18n

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/my-learning-analytics#353]
It was mentioned on another ticket that it would be nice if this project had translated data files. While Google Translate or other browser extension could be used, that would make it easier for a local institution to brand the platform and also provide verified translations over machine generated ones.

**Comment by andrew-gardener** [comment]
https://github.com/i18next/react-i18next looks like a good candidate once the react branch is merged in
